lib/backends/sdc/networks.js

```diff
@@ -774,7 +790,21 @@ function addNetworksToContainerPayload(opts, container, payload, callback) {
                         next(findErr);
                         return;
                     }
-                    payload.networks = [ {uuid: network.uuid, primary: true} ];
+                    payload.networks =
+                        [ {ipv4_uuid: network.uuid, primary: true} ];
```

Comment:
I would've asked about prefixlen/netmask but the "network" object's UUID takes care of that.